data/json/items/comestibles/raw_veggy.json

```
@@ -216,6 +216,25 @@
     "flags": [ "RAW" ],
     "vitamins": [ [ "iron", 8 ] ]
   },
+  {
+    "type": "COMESTIBLE",
+    "id": "lotus_rhizome",
+    "name": { "str": "lotus rhizome" },
```

```suggestion
    "name": { "str": "lotus root" },
```
The common name for this in English is just "lotus root"